rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupportImpl.java

```diff
@@ -45,37 +45,46 @@ public Optional<Coin> getLockingCap() {
     }
 
     @Override
-    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) {
+    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) throws LockingCapIllegalArgumentException {
+        final String INCREASE_LOCKING_CAP_TAG = "increaseLockingCap";
+
+        if (newLockingCap.getValue() <= 0) {
+            String message = "Locking Cap must be greater than zero. Value attempted: %d";
+            String baseMessage = String.format(message, newLockingCap.value);
```

Comment:
```suggestion
            String message = String.format("Locking Cap must be greater than zero. Value attempted: %d");
```
No big deal, but I don't really see the advantage in having 2 variables

Reply:
Done!